src/main/java/thePackmaster/cards/pinnaclepack/StoneSwordCommonAttack.java

```diff
@@ -0,0 +1,39 @@
+package thePackmaster.cards.pinnaclepack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.DexterityPower;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class StoneSwordCommonAttack extends AbstractPinnacleCard {
```

Comment:
*Stone Sword*
The overall design of this card violates too many design principles for it to be in the pack. As it stands, this card will typically have one of two states -- unusable (in all the situations where you can't play it because its a 3-cost card with minimal immediate impact) or game-breaking (when you have Snecko Eye or other cost reduction that lets you abuse the non-exhausting common-rarity dexterity increase on an attack to rapidly outscale enemies).

The best course of action here is to remove this card and replace it with a new card with a new design. I recommend that you create something that is well within the guidelines for complexity.